doc/_includes/tutorials/common/samples/command-line.md

```diff
@@ -11,7 +11,7 @@ The dialog that appears will provide you with a command that you can **copy** an
 Copy and paste the command into a terminal window and use the username and password that you use to log in to IRIDA:
 
 ```bash
-[user@waffles ~]$ ngsArchive.pl -p 4 -s 51
+[user@waffles ~]$ ngsArchiveLinker.pl -p 4 -s 51
```

Comment:
Could you also update this screenshot in the documentation <https://github.com/phac-nml/irida/blob/development/doc/images/tutorials/common/samples/command-line-linker-dialog.png>?